src/pages/index.js

```diff
@@ -0,0 +1,3 @@
+import { Home } from "./home/Home";
```

Comment:
- No need to import ```Home``` for exporting.
- We can directly export it as

```suggestion
export { Home } from "./home/Home";
```